hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -580,6 +580,7 @@ protected void autoArchiveOnCommit(HoodieTable table) {
       return;
     }
 
+    table.getMetaClient().reloadActiveTimeline();
```

Comment:
Why do we need this reload ? Can we avoid it ?

Reply:
Just to make archive happen more timely, I will remove it.